client/src/main/scala/spatutorial/client/services/SPACircuit.scala

```diff
@@ -212,14 +214,22 @@ class AirportConfigHandler[M](modelRW: ModelRW[M, Pot[AirportConfig]]) extends L
       val call: Future[AirportConfig] = AjaxClient[Api].airportConfiguration().call()
       log.info(s"Airport config from server: ${call}")
 
+
       updated(Pending(), Effect(call.map(UpdateAirportConfig)))
     case UpdateAirportConfig(configHolder) =>
       log.info(s"Received airportConfig ${configHolder}")
       log.info("Subscribing to crunchs for terminal/queues")
 
       val crunchRequests: Seq[Effect] = for {tn <- configHolder.terminalNames
                                              qn <- configHolder.queues
-      } yield Effect(Future(GetLatestCrunch(tn, qn)))
+      } yield {
```

Comment:
I suspect this is a strange, direct (?) side-effect in the circuit, is this the right place? Maybe.